Kitodo/src/main/java/de/sub/goobi/forms/ProzesskopieForm.java

```diff
@@ -930,12 +929,10 @@ public String createNewProcess() throws ReadException, IOException, PreferencesE
                 /* Rdf-File schreiben */
                 serviceManager.getFileService().writeMetadataFile(this.rdf, this.prozessKopie);
 
-                /*
-                 * soll der Prozess als Vorlage verwendet werden?
-                 */
-                if (this.useTemplates && this.prozessKopie.isInChoiceListShown()) {
+                // TODO: what does tis if?
```

Comment:
In the past you can define a process as a template for creating new processes with existing properties. If this should work not anymore or should be removed why are you commenting this out instead of removing it?